django/template/defaulttags.py

```diff
@@ -6,6 +6,7 @@
 from datetime import datetime
 from itertools import cycle as itertools_cycle
 from itertools import groupby
+from typing import Iterable
```

Comment:
[`typing.Iterable`](https://docs.python.org/3/library/typing.html#typing.Iterable) has been deprecated since Python 3.9...

```suggestion
from collections.abc import Iterable
```

Reply:
Arg! Wish they would make their mind up.

No problem with the short feedback loop, it's keeping me busy!